apps/codeforafrica/content/pages/hero.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Hero would be an `Object` [widget](https://www.netlifycms.org/docs/widgets/#object) inside an index page, no?